plugins.json

```diff
@@ -2668,6 +2668,36 @@
     },
     "updated_at": "2018-12-18T07:44:40Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-welcomebot",
+    "icon_data": "",
+    "download_url": "https://github.com/mattermost/mattermost-plugin-welcomebot/releases/download/v1.2.0/com.mattermost.welcomebot-1.2.0.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-welcomebot/releases/tag/v1.2.0",
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl7ROGgACgkQ0bVLR6XO/sQjhRAAndPX83MNxx2bLc0LUfai/lPf2NkH2hvU0mcUMKOYKfBsoJrFkmxrYklLeS7sxC7mtSsP2Kv9Ihp3YfBUfLqHZ7xjLrc4WS2q8Cvv7HgL77lFTiV+6bo+GjgoexJi+dILGbcAw4c5n8hqH0++EneeqnHCmwrBIM3LSw0+LN37huf9bVtmZAFtQBDviSndGY3TNf7ZfTL74eK6mQhbU9zVsb/wZHHTlJP0S30TkQZkVrFTXESjW/CqGsftqrqGfVUvDlPw6dGkcKD3yzQ1H4+rBVyudBQMwRNMqmvgIKGIB2BEFi3gbal8nrs/pBZhGHdZfY/wZBfaZY+XykVqoqT9uPabiETbO3/yA5CoEZnmlFNBoUMv+N7EBv58Q7r4+Ko1VO8VSAsHEcHaTpH+NGVNmIwaMu2zRhEEicxpUqCe2i1NDVhq0Er5bKv/vrtLfJ81g9ALSfcnCkNKo5lZt6/zJDOY8JER5ODJbwwgfsezHMH+lqUCPGBWzWsQOzT74IGyCFgh8M0DNuym0P66K6dbNatg+1JY8Pse+XxgddTdlhQfFuUfk0b1bPKu3sgG4/HJe/dvfr51Nu3dvwWlU6NY/wM+VLSb3jpwM2ha6/ch+mbwjT/qF9/uGyNtljEl2xfD9jHHCjtczF+kNhYyA0l6MsCaU4bT/LoJxftmLuBFDlI=",
+    "manifest": {
+      "id": "com.mattermost.welcomebot",
+      "name": "Welcome Bot",
+      "description": "This plugin adds a WelcomeBot that helps add new users to channels.",
```

Comment:
```suggestion
      "description": "WelcomeBot guides new Mattermost users to channels they should join based on their interests",
```

Reply:
I'm not entirely sure that this is a good change:
1. This change is only visible in the Marketplace UI. The admin console actually does show the old description.
![Uploading Screenshot from 2020-06-02 21-25-06.png…]()
2. The text is cut off in the Marketplace.
![Screenshot from 2020-06-02 21-25-15](https://user-images.githubusercontent.com/16541325/83561206-bfdd3e80-a517-11ea-9d5d-6f2287e7d1e2.png)

I'm wondering if the discussion about `description` should take place in a different PR and this PR goes like in like it is.

cc @aaronrothschild

Reply:
@hanzei, that makes sense because we aren't changing the description in the plugin.json in the actual plugin.  I am good with reverting for the following reasons:
* we have to remember to changes this text for every subsequent welcomebot release to marketplace.
* we should not have to manually do this step